plugins/woocommerce/client/admin/client/settings-payments/test/settings-payments-methods.test.tsx

```diff
@@ -1,55 +0,0 @@
-/**
```

Comment:
This test file has been deleted since it only tests for track events. I think we'll need new track events for this new modal experience. But until we decide, I removed it for now.